src/Tools/GenerateProps/targetgroups.props

```diff
@@ -74,6 +74,9 @@
     </TargetGroups>
     <TargetGroups Include="netstandard">
       <NuGetTargetMoniker>.NETStandard,Version=v2.0</NuGetTargetMoniker>
+      <!-- By default, netstandard libraries will be held to the UWP compatibility bar;
+           set this to false in the project to force otherwise. -->
+      <EnableWinRT>true</EnableWinRT>
```

Comment:
> I can rename it to eg `<UWPCompatible>`

This sound much better to me.

>  If the distinction is not interesting 

I do not think the distinction is interesting. Also, I would love to see `BlockReflectionAttribute` to go away completely from corefx (https://github.com/dotnet/corefx/issues/16022).